pkg/activator/net/revision_backends.go

```diff
@@ -63,6 +63,7 @@ type revisionDestsUpdate struct {
 const (
 	probeTimeout   time.Duration = 300 * time.Millisecond
 	probeFrequency time.Duration = 200 * time.Millisecond
+	probePath                    = "/_internal/knative/activator/probe"
```

Comment:
Some random path for activator to probe. It can be changed as @tcnghia has mentioned [here](https://github.com/knative/serving/issues/5918#issuecomment-572790069)

Reply:
This seems to be constant here, which is fine.
It needs to be documented as well.

Reply:
Though it seems quite long for a random probe path.

Reply:
We can follow up to extract this to a configurable option in the future.

Reply:
Yes, please fine an issue, so it's not forgotten